src/catchup/CatchupManager.h

```diff
@@ -18,8 +18,20 @@ namespace stellar
 {
 
 class Application;
-struct HistoryArchiveState;
-struct LedgerHeaderHistoryEntry;
+class LedgerHeaderHistoryEntry;
+
+// Verification mode for downloaded ledgers. VERIFY_BUFFERED_LEDGERS means
+// that ledgers download from history will be compared with ledgers buffered
+// in LedgerManager during catchup, and last downloaded ledger have to be
+// present in that buffered list.
+//
+// When doing manual or command line catchup use DO_NOT_VERIFY_BUFFERED_LEDGERS
+// as buffered ledgers in LedgerManager are not available then.
+enum class VerifyLedgerMode
```

Comment:
I am not sure I understand the change, was it that you didn't like the name of the boolean? Use of enums of this case means increase cognitive load (I have to look up the values of the enum).

Reply:
The name `manualCatchup` does mean much in context of VerifyLedgerChainWork. Why should veryfing ledgers care about that? Using enum with good names in VerifyLedgerChainWork provides information how it will do that verification.

I think that the best solution could be to pass a lambda to VerifyLedgerChainWork that does some extra verification (calls mApp.getLedgerManager().verifyCatchupCandidate(curr);) or even move that last bit of verification outside of that class (but that will clash with #1244).

Reply:
right, what I am suggesting is renaming `manualCatchup` to something more meaningful instead of using an enum. `verifyWithBufferedLedgers` for example. going with a lambda at this point doesn't seem justified.